Snakefile

```diff
@@ -143,18 +143,18 @@ if config["enable"].get("download_osm_data", True):
 
 rule clean_osm_data:
     input:
-        cables="resources/osm/raw/africa_all_raw_cables.geojson",
-        generators="resources/osm/raw/africa_all_raw_generators.geojson",
-        lines="resources/osm/raw/africa_all_raw_lines.geojson",
-        substations="resources/osm/raw/africa_all_raw_substations.geojson",
+        cables="resources/osm/raw/all_raw_cables.geojson",
+        generators="resources/osm/raw/all_raw_generators.geojson",
+        lines="resources/osm/raw/all_raw_lines.geojson",
+        substations="resources/osm/raw/all_raw_substations.geojson",
         country_shapes="resources/shapes/country_shapes.geojson",
         offshore_shapes="resources/shapes/offshore_shapes.geojson",
         africa_shape="resources/shapes/africa_shape.geojson",
     output:
-        generators="resources/osm/clean/africa_all_generators.geojson",
-        generators_csv="resources/osm/clean/africa_all_generators.csv",
-        lines="resources/osm/clean/africa_all_lines.geojson",
-        substations="resources/osm/clean/africa_all_substations.geojson",
+        generators="resources/osm/clean/all_generators.geojson",
```

Comment:
I'd be prone to rename the outputs as "all_clean_{feature}.{ext}" alike the raw version.
What do you think?

Reply:
Let's keep it as is? I like your suggestion as it might reduce a bit code in the Snakefile. Small disadvantages might arise. E.g. code is less explicit, loosing left side assigned parameter which requires different read in in the scripts (.py). E.g. snakemake.inputs[1] instead of snakemake.inputs.generators ... I am sure some can be tackled.

Probably not necessary atm?

Reply:
Ah wait my suggestion was very stupid, keep everything as is but simply rename "all_generators" into "all_clean_generators", "all_lines" into "all_clean_lines" and so on.
No wildcards or anything like that.

When running some codes, having the raw/clean duality also in the name may be handy rather than raw/none, both for symmetries than coding (I had to do some special ifs in the add_scenario PR indeed)

Reply:
Good point! Done